src/main/java/dog/pawbook/logic/parser/FindCommandParser.java

```diff
@@ -27,6 +28,8 @@ public FindCommand parse(String args) throws ParseException {
         }
         String[] nameKeywords = trimmedArgs.split("\\s+");
         List<String> keywords = Arrays.asList(nameKeywords);
+
+        assert (!keywords.isEmpty());
```

Comment:
Makes more sense check this within `NameComtainsKeywordsPredicate`

Reply:
Edited.